tools/release/release_utils.sh

```diff
@@ -66,6 +66,24 @@ function publishImages {
         echo "push image: $image"
         docker tag $image:$baseTag $image:$curTag
         retry 5 docker push $image:$curTag
+
+    done
+    docker logout
+}
+
+function publishPackages{
+    local curTag=$1
+    local baseTag=$2
+    local dockerHubUser=$3
+    local dockerHubPassword=$4
+    docker login ghcr.io -u $dockerHubUser -p $dockerHubPassword
+    local images="$(docker images --format "{{.Repository}}" --filter=reference='goharbor/*:'$baseTag'')"
+    for image in $images
+    do
+        echo "push image: $image"
+        docker tag $image:$baseTag "ghcr.io/"$image:$curTag
+        retry 5 docker push "ghcr.io/"$image:$1curTag
+        
```

Comment:
Is this line break based on some kind of standard? If it is not, please remove it.